R/benchmark.R

```diff
@@ -87,6 +89,7 @@
 #' @return A `Benchmark` object containing these functions
 #' @export
 Benchmark <- function(name,
+                      version,
```

Comment:
Oh, and, do we want to have this as an argument here or something that's set inside the body of the benchmarking code? We don't expect that we will support both versions such that you can select one or the other, right?

Reply:
I didn't give it a default because I wanted the version to necessarily live in each benchmark so it's more visible when it should be updated. It's an extra line, but it feels worth it? Otherwise we (at least I) will likely forget to change these when we alter benchmarks